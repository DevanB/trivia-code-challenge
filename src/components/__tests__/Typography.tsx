import {render} from '@testing-library/react'
import * as React from 'react'
import {H1, P} from '../Typography'

describe('<H1/>', () => {
  it('should render without crashing', () => {
    const {container} = render(<H1 />)
    expect(container).toMatchSnapshot()
  })
})

describe('<P/>', () => {
  it('should render without crashing', () => {
    const {container} = render(<P />)
    expect(container).toMatchSnapshot()
  })
})
