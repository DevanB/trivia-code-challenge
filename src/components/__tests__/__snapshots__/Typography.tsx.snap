// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<H1/> should render without crashing 1`] = `
<div>
  <h1
    class="Typography__H1-n3ni2d-0 hhYBab"
  />
</div>
`;

exports[`<P/> should render without crashing 1`] = `
<div>
  <p
    class="Typography__P-n3ni2d-1 fVZwR"
  />
</div>
`;
